### PR TITLE
Add some elliptic AMR projectors

### PR DESCRIPTION
**Pull request by nilsvu**
### Code review checklist

- [ ] The code is documented and the documentation renders correctly. Run
  `make doc` to generate the documentation locally into `BUILD_DIR/docs/html`.
  Then open `index.html`.
- [ ] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).
- [ ] The PR lists upgrade instructions and is labeled `bugfix` or
  `new feature` if appropriate.

### CODE REVIEW

File: .clang-tidy [hunk 1]
```diff
@@ -76,6 +76,8 @@ Checks: '*,
 -readability-magic-numbers,
 # Same as llvm-qualified-auto above.,
 -readability-qualified-auto,
+# Access specifies can be useful to structure code,
```
Comment (kidder) [inline .clang-tidy]: specifies -> specifiers

**Comment by nilsvu** [comment]
@kidder I made this PR very small now, and will open follow-ups later. Could you review this please?

**Review by kidder — CHANGES_REQUESTED** [review]
you should test the projectors

**Comment by nilsvu** [comment]
@kidder Updated and added a test for p-AMR in action. h-AMR isn't supported by the action-testing framework yet as we discussed, so I can enable that once we have added support.

Please ignore the first two commits (cherry-picked from #5163 and #5781).

**Review by kidder — DISMISSED** [review]
needs a rebase

**Comment by nilsvu** [comment]
Rebased. Thanks for reviewing.

**Review by kidder — APPROVED** [review]

**Pull request merged**